localcoinbot.py

```diff
@@ -314,12 +334,202 @@ def gas(update, context):
     except TypeError as e:
         LOGGER.warning(f'Rate Limited - {str(e)}')
         message = "Sorry Unable to Fetch Gas Estimates Right Now..."
-        gas_msg = context.bot.send_message(
-            chat_id=update.message.chat.id,
+        gas_message = context.bot.send_message(
+            chat_id=update.effective_message.chat.id,
             text=message,
             parse_mode=ParseMode.HTML
             )
-        cleaner(context, gas_msg)
+        cleaner(context, gas_message)
+
+
+def price(update, context):
+    # Gather and Show Crypto Stats
+    remove_command(context, update)
+    price_resp = cg.get_price(
```

Comment:
Just FYI, because this is a request to an outside service I'd start the try-except clause beforehand. In this case I'm sure the CG library has some built in handling though